samples/tfm_integration/psa_protected_storage/sample.yaml

```diff
@@ -0,0 +1,22 @@
+sample:
+  description: Protected Storage API sample
+  name: PSA Protected Storage
+common:
+  tags: psa
+  platform_allow: mps2_an521_nonsecure v2m_musca_s1_nonsecure
+                    nrf5340dk_nrf5340_cpuappns nrf9160dk_nrf9160ns bl5340_dvk_cpuappns
```

Comment:
Going to test on BL5340 or perhaps @lairdjm could do :)

Reply:
Have tested and it builds and runs on the BL5340, output:
```
*** Booting Zephyr OS build v2.6.0-rc1-61-gaadb337a7e20  ***
Protected Storage sample started. PSA Protected Storage API Version 1.0
Writing data to UID1: The quick brown fox jumps over the lazy dog
Info on data stored in UID1:
- Size: 44
- Capacity: 0x200400d0
- Flags: 0x 0
Read and compare data stored in UID1
Data stored in UID1: The quick brown fox jumps over the lazy dog
Overwriting data stored in UID1: Lorem ipsum dolor sit amet
Removing UID1
Writing data to UID2 with overwrite protection: The quick brown fox jumps over the lazy dog
Attempting to write 'Lorem ipsum dolor sit amet' to UID2
Got expected error (PSA_ERROR_NOT_PERMITTED) when writing to protected UID
```

Reply:
@andvib this is the expected output, correct?

Reply:
Looks correct!